modules/exploits/multi/http/getsimplecms_unauth_code_exec.rb

```diff
@@ -0,0 +1,175 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "GetSimpleCMS Unauthenticated RCE",
+      'Description'    => %q{
+        This module exploits a vulnerability found in GetSimpleCMS,
+        which allows unauthenticated attackers to perform Remote Code Execution.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'truerandom (twitter.com/truerand0m) @ Khalifazo' #
```

Comment:
URLs and twitter handlers are generally not permitted in module author metadata. Move any additional information to a trailing comment.

```suggestion
          'truerand0m' # Discovery, exploit and Metasploit
```